### ISSUE TITLE
Could you please tell how to set the hyparameters of the GLUE?

### ISSUE THREAD

**Issue by leoozy** [princeton-nlp/DinkyTrain#8]
Hello, I noticed that you gave a search space of hyparameters on GLUE dataset. I am confused that how you search the hyparameters. Did you train each task on each of the hyparameters with different seeds? There are about fifty combinations of parameters. Did you finetune GLUE with each of the parameters? Thank you.

**Comment by leoozy** [comment]
The speach space refers to the search space in Should You Mask 15% in Masked Language Modeling?

**Comment by CodeCreator** [comment]
Yes, we indeed run a large number of fine-tuning runs. We have a grid of hyperparameters for each task (see Appendix A.2) and run three random seeds per run.
